test/e2e/handler/nns_update_timestamp_test.go

```diff
@@ -28,22 +26,28 @@ var _ = Describe("[nns] NNS LastSuccessfulUpdateTime", func() {
 			originalNNSs[node] = nodeNetworkState(key)
 		}
 	})
-	Context("when network configuration hasn't change", func() {
-		It("should not be updated", func() {
+	Context("when network configuration is not changed by a NNCP", func() {
+		FIt("should be updated only if configuration changed", func() {
+			timeout := 2 * nmstatenode.NetworkStateRefresh
+			By(fmt.Sprintf("Sleeping %s to wait for nns refresh", timeout))
+			time.Sleep(timeout)
 			for node, originalNNS := range originalNNSs {
-				// Give enough time for the NNS to be updated (3 interval times)
-				timeout := 3 * nmstatenode.NetworkStateRefresh
+				// Give enough time for the NNS to be reconcile(1.5 interval times)
 				key := types.NamespacedName{Name: node}
-
-				Consistently(func() shared.NodeNetworkStateStatus {
-					return nodeNetworkState(key).Status
-				}, timeout, time.Second).Should(MatchAllFields(Fields{
-					"CurrentState":             WithTransform(shared.State.String, Equal(originalNNS.Status.CurrentState.String())),
-					"LastSuccessfulUpdateTime": Equal(originalNNS.Status.LastSuccessfulUpdateTime),
-					"Conditions":               Equal(originalNNS.Status.Conditions),
-				}), func() string {
-					return fmt.Sprintf("currentState diff: \n%s", diff.LineDiff(originalNNS.Status.CurrentState.String(), nodeNetworkState(key).Status.CurrentState.String()))
-				})
+				currentStatus := nodeNetworkState(key).Status
+				originalStatus := originalNNS.Status
+				if currentStatus.CurrentState.String() != originalStatus.CurrentState.String() {
```

Comment:
Isn't this given? There is nothing applying NNCP, so this should stay intact. And it is not in the scope of the test t check that current status stays the same, is it?

Reply:
Looks like network configuration changes without human intervention at kubevirtci, some ipv6 routes disappear sometimes, also I have see at openshift it happends a lot when I was debugging the big cluster, so the tests is prepare for it at the end what we want is to test that updatetimestamp has the expected value.

Reply:
I'm not sure I understand the test there. It tests two thing and it depends on some random changes on the host which tests gets triggered?

Reply:
I have splitet the test as @rhrazdil suggested, please take a look